lms/static/scripts/frontend_apps/components/ContentSelector.tsx

```diff
@@ -200,6 +206,15 @@ export default function ContentSelector({
     onSelectContent({ type: 'url', url: file.id });
   };
 
+  const selectMoodlePage = (page: File | Page) => {
+    cancelDialog();
+    onSelectContent({
+      type: 'url',
+      url: page.id,
+      name: `Moodle page: ${page.display_name}`,
+    });
+  };
```

Comment:
We clearly need to give this component a thought and remove all duplicated stuff somehow 😅